packages/zmq-async/zmq-async.5.1.3/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "anders@fugmann.net"
+authors: ["Rudi Grinberg"]
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+doc: "https://issuu.github.io/ocaml-zmq/"
+dev-repo:
+  "git+https://github.com/issuu/ocaml-zmq.git"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "zmq" {= version}
+  "dune"
+  "async_unix" {>= "v0.11.0" & < "v0.13"}
+  "async_kernel" {>= "v0.11.0" & < "v0.13"}
+  "base" {>= "v0.11.0" & < "v0.13"}
```

Comment:
Is this really not compatible with async and base v0.13.0?

Reply:
I just tried building locally and it seemed fine, removed that constraint in e133620.